src/VisualStudio/Core/Impl/ProjectSystem/CPS/CPSProjectFactory.cs

```diff
@@ -203,11 +151,13 @@ public async Task<IWorkspaceProjectContext> CreateProjectContextAsync(Guid id, s
         const string itemName = BuildPropertyNames.IntermediateAssembly;
 
         var values = data.GetItemValues(itemName);
-        if (values.Length != 1)
+        if (values.Length > 1)
+        {
+            var joinedValues = string.Join(";", values);
+            throw new InvalidProjectDataException(itemName, joinedValues, $"Item group '{itemName}' is required to specify a single value: '{joinedValues}'.");
+        }
+        else if (values.Length == 0)
```

Comment:
It seems a bit suspicious to me that if we didn't get any back we'd throw an exception since that could be an error case...so going to change the logic a bit.